mix.exs

```diff
@@ -39,8 +45,10 @@ defmodule Recaptcha.Mixfile do
     [
       {:httpoison, "~> 0.11.0"},
       {:poison, "~> 1.5 or ~> 2.0 or ~> 3.0"},
+
```

Comment:
This line serves me quite well to separate actual requirements from the development packages.